### ISSUE TITLE
Linq2db support

### ISSUE THREAD

**Issue by jogibear9988** [Giorgi/EFCore.Visualizer#16]
Maybe also linq2db orm could be supported?
Is the https://github.com/Giorgi/EFCore.Visualizer/blob/main/src/IQueryableObjectSource/EFCoreQueryableObjectSource.cs the only file directly using EFCore?

**Comment by Giorgi** [comment]
Yes, that is the only file specific to EF Core. Does linq2db have a way to get generated SQL from a query?
